msteams-platform/resources/dev-preview/developer-preview-intro.md

```diff
@@ -2,31 +2,41 @@
-
 >[!NOTE]
 >Features included in preview may not be complete, and may undergo changes before becoming available in the public release. They are provided for testing and exploration purposes only. They should not be used in production applications.
 
+Developer Preview is a public program for developers which provides early access to unreleased features in Microsoft Teams. This allows you to explore and test upcoming features for potential inclusion in your Microsoft Teams app. We also welcome [feedback](~/feedback.md) on any feature in developer preview. Developer preview is enabled per Microsoft Teams client, so you don't need to worry about affecting your entire organization.
+
 ## Developer preview app manifest
 
 Many features enabled in developer preview will require alterations to your app manifest JSON file. To do so you'll need to use the [developer preview manifest schema](~/resources/schema/manifest-schema-dev-preview.md) If you use this schema, you will not be able to use [App Studio](~/get-started/get-started-app-studio.md) to make these changes, nor will you be able to use it to upload your app for testing. To upload your app you'll need to click the `More apps` icon on the app bar, then select the `Upload a custom app link`. Using this method you can only upload a zipped version of your app package.
 
->[!NOTE]
->You may find it useful to use App Studio to create the non-developer preview portions of your app package, then export that package and manually edit the `manifest.json` file to add the developer preview features you wish to use. Once you've added developer preview features to the `manifest.json` file you will not be able to re-import the package into App Studio.
+You may find it useful to use App Studio to create the non-developer preview portions of your app package, then export that package and manually edit the `manifest.json` file to add the developer preview features you wish to use. Once you've added developer preview features to the `manifest.json` file you will not be able to re-import the package into App Studio.
 
 ## Enable developer preview
 
->[!NOTE]
->Developer preview is enabled on a per-client basis. Uploading an app that uses developer preview features may cause clients that have not enabled developer preview to behave unexpectedly.
+Developer preview is enabled on a per-client basis, but the option to turn on developer preview is controlled at the organization level. Turn on the setting for enabling uploading of apps in the admin console of your tenant as described [here](~/get-started/get-started-tenant.md).
+
+Using an app that contains developer preview features may cause clients that have not enabled developer preview to behave unexpectedly. If you do not see an entry for developer preview the most likely reason is your organization is not configured for app uploading.
+
+### On a desktop or web client
```

Comment:
Suggesting for this change to explicitly call out enabling admin console settings:


 ### On a desktop or web client
 To enable the public developer preview on a desktop or web client, you need to do the following:

 1. Ensure the setting for enabling uploading of apps is enabled in the admin console of your tenant as described [here](~/get-started/get-started-tenant.md).
1. Click on your profile (either in the upper right or lower left of the Teams interface) to display the Teams menu.
1. Select About → Developer preview.
1. Select **Switch to Developer preview**.